great_expectations/validator/validator.py

```diff
@@ -148,7 +151,7 @@ def __init__(
         self._validator_config = {}
 
         if batches is None:
-            batches = tuple()
+            batches = []
```

Comment:
That is correct! The batches arg in the constructor is only ever used in `load_batch_list()` (which specifies that it needs a `List[Batch]`. If the user does not provide a `batch` arg, we should create an empty list as opposed to a tuple. 

There should be no change in behavior since we don't use tuple-specific actions (`load_batch_list()` does a simple iteration so any `Iterable` obj should suffice).